docs/content/en/docs/install/install.md

```diff
@@ -160,3 +160,29 @@ metadata:
 
 You see the annotation line `keptn.sh/lifecycle-toolkit: "enabled"`.
 This annotation tells the webhook to handle the namespace.
+
+You could instead use `labels` with this same key.
```

Comment:
there was a misunderstanding in the conversation on slack, for the namespace we support ONLY annotation, this sentence is a very important reflection, but is valid only for the annotations/labels used on app/workload

Reply:
Ooh, thanks for clearing that up!  I deleted this line as well as the ensuing information about the cool things you could do were this a label ;-)